CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 ## Unreleased
 
+- `sdk.alerts.search_all_pages()` to retrieve all alert pages.
```

Comment:
You need to put these under a header, such as `Added`.
There is already an `Added` section below.

Reply:
Moved.